### ISSUE TITLE
View with UNION ALL crashes the server on startup

### ISSUE THREAD

**Issue by tobia** [yandex/ClickHouse#2208]
I'm using the latest ClickHouse 1.1.54370 on Ubuntu Trusty.

If you create a simple view with UNION ALL:

```
:) create view test_union as select 1 as test union all select 2;

CREATE VIEW test_union AS
SELECT 1 AS test
UNION ALL
SELECT 2

Ok.
```

It seems to work:

```
:) select * from test_union;

SELECT *
FROM test_union 

┌─test─┐
│    1 │
└──────┘
┌─test─┐
│    2 │
└──────┘

2 rows in set. Elapsed: 0.015 sec. 
```

***But the metadata file is created with escape characters in it:***

```
ATTACH VIEW test_union
(
    test UInt8
) AS
SELECT 1 AS test
^[[1mUNION ALL^[[0m
SELECT 2
```

And subsequent restarts of the server crash with an error:

```
[ 17 ] <Error> Application: DB::Exception: Cannot create table from metadata file /var/lib/clickhouse/metadata/default//test_union.sql, error: DB::Exception: Syntax error (in file /var/lib/clickhouse/metadata/default//test_union.sql): failed at position 63 (line 6, col 1): ^[[1mUNION ALL^[[0m
SELECT 2
. Unrecognized token, stack trace:
0. /usr/bin/clickhouse-server(StackTrace::StackTrace()+0x15) [0x82e3db5]
1. /usr/bin/clickhouse-server(DB::Exception::Exception(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, int)+0x21) [0x2cc1371]
2. /usr/bin/clickhouse-server(DB::parseQuery(DB::IParser&, char const*, char const*, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)+0xa7) [0x82acce7]
3. /usr/bin/clickhouse-server(DB::createTableFromDefinition(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, DB::Context&, bool, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)+0x6b) [0x8048cab]
4. /usr/bin/clickhouse-server() [0x8045615]
5. /usr/bin/clickhouse-server() [0x8045c82]
6. /usr/bin/clickhouse-server(ThreadPool::worker()+0x19d) [0x857af4d]
7. /usr/bin/clickhouse-server() [0x8d9675e]
8. /lib/x86_64-linux-gnu/libpthread.so.0(+0x8183) [0x7f5ac0ba7183]
9. /lib/x86_64-linux-gnu/libc.so.6(clone+0x6c) [0x7f5ac05ce03c]
```

**Comment by alexey-milovidov** [comment]
Fixed in master: 669b0d4b

**Comment by alexey-milovidov** [comment]
Thank you for the report!
